src/sbd-inquisitor.c

```diff
@@ -1058,7 +1058,7 @@ int main(int argc, char **argv, char **envp)
 			break;
 		case '1':
 			timeout_watchdog = atoi(optarg);
-                        if(timeout_watchdog > 5) {
+                        if (timeout_watchdog_warn_is_default && timeout_watchdog > 5) {
                             timeout_watchdog_warn = (int)timeout_watchdog / 5 * 3;
```

Comment:
Maybe we should move that to after the parameter handling so that we don't have different behaviour depending on the -1 -5 order.

Reply:
Forget that - working in both orders.
But we could still do the calculation for all cases afterwards.